config/initializers/permissions_policy.rb

```diff
@@ -0,0 +1,11 @@
+# Define an application-wide HTTP permissions policy. For further
```

Comment:
Again. Seems no point keeping a file if it is not being used.

Reply:
yeah; I'll remove it.

Reply:
Yes, we can remove this.